README.md

```diff
@@ -113,16 +117,24 @@ Therefore, this part of portfolio is not customizable. But don't worry we have a
   - You can define new theme similarly as other themes and you can assign name of that new defined theme to `chosenTheme`.
 - That's it. You just need to change the theme name and the code will take care of everything else.
   
+  Run `npm start` to check if everything is ok.
+  
 # Deployment 📦 
-- Once you have done with your setup and you have successfully completed all steps above, You need to put your website online!
-- I highly recommend to use [Github Pages](https://create-react-app.dev/docs/deployment/#github-pages) to achieve this on the EASIEST WAY.
-- To deploy your website, first you need to create github repository with name `<your-github-username>.github.io`. Please don't give any other name.
-- Push the generated code to the `master` branch of this repository.
-- Now, we need to generate production build and deploy website.
-  - ```npm run build``` command will generate the production build.
-  - ```npm run deploy``` will create another branch called `gh-pages` and it will push the `build` files to that branch.
-- The last step in deploying is to enable `Github Pages` in settings of the repository and select `gh-pages` branch.
-- Now, your website is successfully deployed and you can visit it at `<your-github-username>.github.io`.
+- Once you are done with your setup and have successfully completed all steps above, you need to put your website online!
+- I highly recommend using [Github Pages](https://create-react-app.dev/docs/deployment/#github-pages) to achieve this the EASIEST WAY.
+- To deploy your website, you have two options. First you need to create a github repository with the name `<your-github-username>.github.io`. Please don't give it any other name.
+- Now, you need to generate a production build and deploy the website.
+
+Option 1:
+  - Run ```npm run build``` to generate the production build folder.
+  - Enter the build folder, `git init` and push the generated code to the `master` branch of your new repository. That's it. Done. 
+   You may need to `git init` and force push at every new build.
+   
+Option 2 (will not work with [user pages](https://docs.github.com/en/github/working-with-github-pages/about-github-pages)):
```

Comment:
`**Option 2 (will not work with [user pages](https://docs.github.com/en/github/working-with-github-pages/about-github-pages)):**`